### PR TITLE
New Admin Landing Page

### PR DESCRIPTION
**Pull request by dereksmart**
<strong>Instructions (New Site Flow): </strong>
- Before you do anything, go to the Jetpack admin page and click "RESET EVERYTHING" at the bottom of the page. 
- Checkout to branch `git checkout enhancement/nux-landing-dev`
- <strong>`grunt` (required to see css changes) - `npm up` first if grunt doesn't work</strong>
- Try flow with both Jump Starting and skipping Jump Start
- Play around in the main Jetpack admin page
- Please test the Jump Start flow as well.  Minor changes were made there too.  You can use the "reset everything" link at the footer of the page.  
- VaultPress CTA's will be different based on the status of VP.  (activated/deactivated/registered...)
- Test responsive views.

### CODE REVIEW

File: _inc/jp.js [hunk 1]
```diff
@@ -84,11 +106,16 @@
 				}, 100 );
 			}
 		};
+
+		//This function will track the number of clicks on the "See the other X Jetpack features"
+		$( '.full-features-btn').click(function(){
```
Comment (samhotchkiss) [inline _inc/jp.js]: missing a space

File: _inc/lib/admin-pages/class.jetpack-landing-page.php [hunk 2]
```diff
@@ -272,7 +291,9 @@ function page_admin_scripts() {
 				'jumpstart_modules' => $this->jumpstart_module_tag( 'Jumpstart' ),
 				'show_jumpstart'    => $this->jetpack_show_jumpstart(),
 				'activate_nonce'    => wp_create_nonce( 'jetpack-jumpstart-nonce' ),
+				'admin_nonce'       => wp_create_nonce( 'jetpack-admin-nonce' ),
 				'jumpstart_stats_urls'  => $this->build_jumpstart_stats_urls( array( 'dismiss', 'jumpstarted', 'learnmore', 'viewed', 'manual' ) ),
+				'nux_admin_stats_urls'  => $this->build_nux_admin_stats_urls( array( 'enabled', 'deactivated', 'learnmore' ) ),
 				'site_url_manage'   => Jetpack::build_raw_urls( get_site_url() ),
 			)
```
Comment (samhotchkiss) [inline _inc/lib/admin-pages/class.jetpack-landing-page.php]: code style-- lines 295/296 aren't lining up

File: views/admin/admin-page.php [hunk 3]
```diff
@@ -7,35 +7,245 @@
...
+					'790618302648bd80fa8a55497dfd8ac8',
+					'6e238edcb0664c975ccb9e8e80abb307',
+					'4e6c84eeab0a1338838a9a1e84629c1a',
+					'9d4b77080c699629e846d3637b3a661c',
+					'4626de7797aada973c1fb22dfe0e5109',
+					'190cf13c9cd358521085af13615382d5',
+				);
+
+				// Pick a random HE.
+				$jetpack_he = array_rand( $jetpack_hes, 1 );
```
Comment (samhotchkiss) [inline views/admin/admin-page.php]: You can drop these two lines, you do this down in L188

File: views/admin/admin-page.php [hunk 4]
```diff
@@ -7,35 +7,245 @@
...
+
+				// Get a random profile URL.
```
Comment (samhotchkiss) [inline views/admin/admin-page.php]: I don't think you meant to have this comment :)

File: views/admin/admin-page.php [hunk 5]
```diff
@@ -7,35 +7,245 @@
...
+
+				$jetpack_contact_url = sprintf(
+					'<a href="http://jetpack.me/contact-support/" target="_blank" title="%1$s">%1$s</a>',
+					esc_attr__( 'contact us directly', 'jetpack' )
+				);
+
+				printf(
+					_x(
+						'We offer free, full support to all of our Jetpack users. Our support team is always around to help you. %1$s, %2$s, or %3$s',
+						'1: View our support page; 2: check the forums for answers, 3: contact us directly',
```
Comment (samhotchkiss) [inline views/admin/admin-page.php]: You build the links about, but you don't actually use them in the string here...

**Comment by samhotchkiss** [comment]
Took a first pass through the code, looks pretty good, there were a few little style issues.  I'll look more closely on Monday

**Comment by samhotchkiss** [comment]
Also, looks like it needs a rebase

File: views/admin/admin-page.php (hunk 3 above)
Comment (jeherve) [inline views/admin/admin-page.php]: Thanks, done in 913810bc0767eb998c9bec8e1989ef6176062a87

File: views/admin/admin-page.php (hunk 4 above)
Comment (jeherve) [inline views/admin/admin-page.php]: Clarified in 913810bc0767eb998c9bec8e1989ef6176062a87

File: views/admin/admin-page.php (hunk 5 above)
Comment (jeherve) [inline views/admin/admin-page.php]: They're used in the string, at the end: `%1$s, %2$s, or %3$s`

Comment (samhotchkiss) [inline views/admin/admin-page.php]: Hah :) I saw the text written out here, and spaced on what was happening.  :beer:

**Comment by samhotchkiss** [comment]
This is amazing.  Awesome work everyone!

The only "issue" I found was that if, before being prompted to Jump Start, I go to dashboard, I get:
image

If I click the activate button, I lose the ability to Jump Start

What do we think?  Worth addressing this scenario?

**Comment by dereksmart** [comment]
I think it's worth addressing, but not in this branch since it's likely been the case since the beginning of Jump Start

**Comment by dereksmart** [comment]
Oh that's the manage CTA - I thought it was the connect CTA. 

That's not an issue.  Jump Start intentionally gets out of the way if any of the modules have been manually configured.

**Pull request merged**